src/i18n/locale-en.json

```diff
@@ -1688,5 +1688,8 @@
     "guide.step_plugin.visual-graph-expand.content": "Double click on <b>{{iriLabel}}</b> to expand the graph.",
     "guide.step_plugin.guide-ended.title": "End of guide",
     "guide.step_plugin.guide-ended.content": "This guide has ended.",
-    "guide.step_plugin.execute-sparql-query.query-not-same.error": "The query is not the same as the guide query!"
+    "guide.step_plugin.execute-sparql-query.query-not-same.error": "The query is not the same as the guide query!",
+    "guide.confirm.cancel.message": "Are you sure you want to stop the guide?",
+    "guide.unexpected.error.message": "The guide was cancelled due to an unexpected error. Please run the guide again and if the problem persists contact the support.",
+    "guide.start.unexpected.error.message": "The guide can be started due to an unexpected error. Please run the guide again and if the problem persists contact the support."
```

Comment:
I think this should be "The gude **cannot** be ...". Also change the second sentence to "Please try running the ...".